backward.hpp

```diff
@@ -2431,17 +2432,25 @@ class TraceResolverLinuxImpl<trace_resolver_tag::libdwarf>
       ELF_GET_DATA(32)
     } else if (e_ident[EI_CLASS] == ELFCLASS64) {
       // libelf might have been built without 64 bit support
-#if __LIBELF64
```

Comment:
This is a bit unrelated, but libelf installed on my PC (it's Arch, but it's true also for Ubuntu 20.04 at least) does not have this definition, so the entire branch is omitted, so no 64-bit data, so no debuglink info. Probably we can work it around in a better way, but this line really stops me from having nice source locations.